config/docker_auth/auth_config.yml

```diff
@@ -0,0 +1,28 @@
+server:
+  addr: ':5001'
+  # TODO: not sure what it's for, the doc also have away for letsencrypt
+  certificate: '/path/to/server.pem'
+  key: '/path/to/server.key'
+
```

Comment:
Would these be the same as the ones in the [`nginx.config`](https://github.com/Seneca-CDOT/docker.cdot.systems/blob/b936d5fbac098178109dd5997d6c3aa9f8d2b293/config/etc/nginx/nginx.conf#L22-L23)?

Reply:
Probably, let try